lib/rules/sort-keys.js

```diff
@@ -37,6 +37,38 @@ function getPropertyName(node) {
     return node.key.name || null;
 }
 
+/**
+ * Check the Line Separated Groups
+ *
+ * - Compare the start line of the current node with the last line of the previous node.
+ *   If the difference between the two values is greater than 0, Not the same group.
+ * @param {number} thisStartLine is start line of the current node.
+ * @param {number} prevEndLine is end line of the previous node.
+ * @param {Array<string>} sourceCodeLines use to check comment between groups.
+ * @returns {boolean} check group
+ * @private
+ */
+function checkLineSeparatedGroups(thisStartLine, prevEndLine, sourceCodeLines) {
+    let blockCommentFlag = false;
+    let commentOffset = 0;
```

Comment:
@yeonjuan @mdjermanovic 

I used the forEach function to check the comment line, but I'm not sure if it's okay.

Is there a better way?